packages/esm-home-app/src/dashboard/home-dashboard.component.tsx

```diff
@@ -25,6 +25,7 @@ export default function HomeDashboard({ match, canSearch }: HomeDashboardProps)
                   <Extension />
                 </div>
               </ExtensionSlot>
+              <ExtensionSlot extensionSlotName="active-visits-widget" />
```

Comment:
This isn't an appropriate slot name. Slot names should describe the _location_ and _possible contents_ of the slot, and be suffixed by `-slot`. Note that `homepage-dashboard-slot` is also badly named; it should be called something like `homepage-buttons-slot`, because it can only be used to render buttons. I would suggest calling your slot something like `homepage-widgets-slot`.

Reply:
In addition I'd say let's not put it in the same container. It should be in a separate container and not be excluded if buttons are disabled.

Maybe have a different config for it to disable widgets...

Reply:
Thanks @brandones ! I'll take care of it from the next time.
Thanks @FlorianRappl I'll separate it from the container.

Reply:
@FlorianRappl , should I make an option in config for disabling the widgets?

Reply:
I'd say yes (for consistency reasons).

Reply:
Sure, I'll add.